php-classes/Slate/Term.php

```diff
@@ -190,6 +190,44 @@ public static function getClosestGraduationYear()
         return $Term->getGraduationYear();
     }
 
+    public static function getClosestConcurrentTermIDs($forceRefresh = false)
+    {
+        $cacheKey = "slate-terms/closest-concurrent-ids";
+
+        if (!$forceRefresh && false !== ($termIds = Cache::fetch($cacheKey))) {
+            return $termIds;
+        }
+
+        try {
+            $termIds = static::getClosest()->getConcurrentTermIDs();
+        } catch (TableNotFoundException $e) {
+            $termIds = [];
+        }
+
+        Cache::store($cacheKey, $termIds);
+
+        return $termIds;
+    }
+
+    public static function getClosestMasterConcurrentTermIDs($forceRefresh = false)
+    {
+        $cacheKey = "slate-terms/closest-master-concurrent-ids";
```

Comment:
@nbey add calls to clear both cache keys after the `parent::save()` call in `Term`'s `save` override